<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Journal_Interface</class>
 <widget class="QDialog" name="Journal_Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Journal_Interface</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Auteurs:</string>
   </property>
   <property name="buddy">
    <cstring>auteurs_journal</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="auteurs_journal">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>9</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>49</y>
     <width>32</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Titre:</string>
   </property>
   <property name="buddy">
    <cstring>titre_journal</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="titre_journal">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>49</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>97</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Année d'édition:</string>
   </property>
   <property name="buddy">
    <cstring>annee_journal</cstring>
   </property>
  </widget>
  <widget class="QSpinBox" name="annee_journal">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>634</width>
     <height>26</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>68</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CODE ISSN:</string>
   </property>
   <property name="buddy">
    <cstring>id_journal</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_journal">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>133</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>94</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom du journal:</string>
   </property>
   <property name="buddy">
    <cstring>nom_journal</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="nom_journal">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>173</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Volume:</string>
   </property>
   <property name="buddy">
    <cstring>vol_journal</cstring>
   </property>
  </widget>
  <widget class="QSpinBox" name="vol_journal">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>213</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>52</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Numéro:</string>
   </property>
   <property name="buddy">
    <cstring>num_journal</cstring>
   </property>
  </widget>
  <widget class="QSpinBox" name="num_journal">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>245</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>33</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Page:</string>
   </property>
   <property name="buddy">
    <cstring>page_journal</cstring>
   </property>
  </widget>
  <widget class="QSpinBox" name="page_journal">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>277</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>176</width>
     <height>28</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>vol_journal</tabstop>
  <tabstop>num_journal</tabstop>
  <tabstop>page_journal</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Journal_Interface</receiver>
   <slot>ValideEnregistrementJournal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Journal_Interface</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onOKajouter()</slot>
  <slot>ValideEnregistrementJournal()</slot>
 </slots>
</ui>
